cfgov/jinja2/v1/about-us/careers/_template-linkedin.html

```diff
@@ -8,7 +8,7 @@ <h2 class="h4">
     Connect with us to stay updated on the work
     we do and new opportunities to be a part of it.
 </p>
-<a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+<a href="/external-site/?ext_url=https://www.linkedin.com/company/consumer-financial-protection-bureau"
    class="jump-link jump-link__external-link jump-link__underline">
```

Comment:
`jump-link__underline` should be unnecessary if you wrap the text inside the link with `<span class="jump-link_text">`, i.e.:

```html
<a href="/external-site/?ext_url=https://www.linkedin.com/company/consumer-financial-protection-bureau"
     class="jump-link jump-link__external-link">
    <span class="jump-link_text">Follow us on LinkedIn</span>
</a>
```

N.B.: It's important that there is no whitespace between `LinkedIn` and the closing `</span>`